mods/_core/file_manager/filemanager.inc.php

```diff
@@ -152,6 +152,7 @@ function preExtractCallBack($p_event, &$p_header) {
 */
 	function preImportCallBack($p_event, &$p_header) {
 		global $IllegalExtentions;
+		$legalExtensions = array("txt", "xml");
```

Comment:
This does not seem right. This would only allow txt & xml files to be packaged. Creating a file extention whitelist is not feasible currently.